lib/features/modeling/BpmnUpdater.js

```diff
@@ -20,6 +20,14 @@ import {
   is
 } from '../../util/ModelUtil';
 
+import {
+  hasExplicitlyBoundedInternalLabel as hasExplicitlyBoundedEmbeddedLabel
```

Comment:
If a utility has no use outside of a single file (which seems to be the case here), leave it inside that single file.

We prevent utility creep this way.